### ISSUE TITLE
service control (service_control_%) and id usage

### ISSUE THREAD

**Issue by AdSchellevis** [opnsense/core#6582]
**Important notices**

Before you add a new report, we ask you kindly to acknowledge the following:

- [x] I have read the contributing guide lines at https://github.com/opnsense/core/blob/master/CONTRIBUTING.md
- [x] I am convinced that my issue is new after having checked both open and closed issues at https://github.com/opnsense/core/issues?q=is%3Aissue

**Is your feature request related to a problem? Please describe.**

When services require an `id` field, only one item is being returned when asking for such a service. For example `service_by_name('openvpn')` would return the first (?) item in the list. It might be practical if omitting the number on a stop/start would treat it as a wildcard and stops/starts all of the items in the group. Within the `xmlrpc` service definition  we do ship services to reconfigure if needed, but now it only reconfigures one (which is also a bit confusing).

Before considering this behavioral change, we might need to discuss this further. 

**Describe the solution you like**

Expand the service control actions which are exposed via `pluginctl -s` to perform the action on a list of services if more than a single one matches the question. 

**Describe alternatives you considered**

None, there might be some downsides on changing this behavior.

**Comment by fichtner** [comment]
I don’t think this is a service_by_name() issue as it only pertains to legacy pages and will die soon enough with those. I don’t even think there is any ID-based service left in legacy code.

If we talk about pluginctl behaviour that can indeed be changed to restart all by default (empty string but not “0” which is a valid ID), but keep in mind the status call tries to return a PID so it would need to return multiple PIDs as well which is to be avoided from MVC end as it would break the return value expectation unless modelled accordingly.
